docs/building-on-etherlink/sandbox.md

```diff
@@ -0,0 +1,77 @@
+---
+title: Running a local sandbox
+---
+
+You can run a local sandbox to test Etherlink transactions and smart contracts locally.
+The sandbox consists of an [EVM node](/network/evm-nodes) in a special mode that includes the functionality of the sequencer and does not need any other nodes.
+The local sandbox is intended for local testing and debugging.
+
+Follow these steps to start the local sandbox:
+
+1. Get the latest release of the `octez-evm-node` binary as described in [Running an Etherlink EVM node](/network/evm-nodes).
+
+1. Initialize the data directory for the EVM node, as in these examples, which use the variable `<EVM_DATA_DIR>` to represent the local folder for the data directory:
```

Comment:
I’d say this step is only useful if the user wants to have an ad-hoc configuration (like enabling websockets).

Otherwise, it can be skipped.

Reply:
Going to leave this in, it's a simple step and only takes a second. It helps the user by letting them know about the data dir and.